packages/compass-settings/src/utils/user-preferences.ts

```diff
@@ -0,0 +1,59 @@
+import { promisifyAmpersandMethod } from 'mongodb-data-service';
+import Preferences from 'compass-preferences-model';
+// import type { THEMES } from 'compass-preferences-model';
+
+export type UserPreferences = {
+  agreedToLicense: boolean;
+  authKerberos: boolean;
+  authLdap: boolean;
+  authMongodb: boolean;
+  authX509: boolean;
+  autoUpdates: boolean;
+  chartView: boolean;
+  currentUserId: string;
+  enableFeedbackPanel: boolean;
+  enableMaps: boolean;
+  id: string;
+  indexDDL: boolean;
+  lastKnownVersion: string;
+  networkTraffic: boolean;
+  queryBuilder: boolean;
+  serverStats: boolean;
+  showAutoUpdateBanner: boolean;
+  showExplainPlanTab: boolean;
+  showedNetworkOptIn: boolean;
+  singleDocumentCrud: boolean;
+  sslAll: boolean;
+  sslServer: boolean;
+  sslUnvalidated: boolean;
+  telemetryAnonymousId: string;
+  theme: string; // todo use THEMES
+  trackErrors: boolean;
+  trackUsageStatistics: boolean;
+};
```

Comment:
Are all of these used? Do we allow to change them?

Reply:
skimmed it down.